lib/features/destination_picker/presentation/destination_picker_view.dart

```diff
@@ -25,17 +35,28 @@ class DestinationPickerView extends GetWidget<DestinationPickerController>
   final _imagePaths = Get.find<ImagePaths>();
   final _responsiveUtils = Get.find<ResponsiveUtils>();
 
-  DestinationPickerView({Key? key}) : super(key: key);
+  @override
+  final controller = Get.find<DestinationPickerController>();
```

Comment:
IMO, `controller` is a built-in GetWidget. Why need to do like that?

Reply:
Since we are not calling the view directly by navigating in `Page` but rendering the view directly to the current screen as `dialog`, the `controller` will be null when we use it. in view's constructor